src/Tribe/JSON_LD/Abstract.php

```diff
@@ -255,12 +262,48 @@ public function register( $post ) {
 		if ( $this->exists( $id ) ) {
 			return self::$posts[ $id ];
 		}
-
 		self::$posts[ $id ] = get_post( $id );
-
 		return self::$posts[ $id ];
 	}
 
+
+	/**
+	 * Public method to have access to the types
+	 *
+	 * @since TBD
+	 *
+	 * @return array
+	 */
+	public function get_types() {
+		return self::$types;
+	}
+
+	/**
+	 * Register the current $type to prevent duplicates entries with different $types and IDs
+	 *
+	 * @since TBD
+	 *
+	 * @param $post
+	 * @param $type
+	 *
+	 * @return mixed
```

Comment:
**no need to change**, just curious -- doesn't this method always `@return` an array?